ssl/ssl_sess.c

```diff
@@ -828,6 +828,13 @@ long SSL_SESSION_set_time(SSL_SESSION *s, long t)
     return (t);
 }
 
+int SSL_SESSION_get_protocol_version(const SSL_SESSION *s)
+{
+    if (s == NULL)
+        return (0);
```

Comment:
Yes, that is inconsistent. As a general rule though we are trying to standardise on not doing the NULL check for the main object of a function, so I think we should remove this. There are other SSL_SESSION functions that don't do this.